src/smart-components/myUserAccess/bundles/openshift.js

```diff
@@ -1,23 +1 @@
-import React, { Fragment } from 'react';
-import { Alert } from '@patternfly/react-core';
-import RhelBundle from './rhel';
-import { useIntl } from 'react-intl';
-import messages from '../../../Messages';
-
-import './MUABundles.scss';
-
-const OpenshiftBundle = (props) => {
-  const intl = useIntl();
-  return (
-    <Fragment>
-      <div className="pf-u-mb-xl">
-        <Alert isInline variant="info" title="OpenShift Clusters permissions are not managed with User Access">
-          <p>{intl.formatMessage(messages.openshiftPermissions)}</p>
```

Comment:
Can you remove this message from `src.Messages.js`? Seems like it's not used anywhere else.

Reply:
@Hyperkid123 done!